sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c

```diff
@@ -363,11 +363,32 @@ static bool _pio_from_iface_addr(gnrc_pktsnip_t **res, gnrc_ipv6_netif_addr_t *a
     return false;
 }
 
+static inline bool _check_prefixes(gnrc_ipv6_netif_addr_t *a, gnrc_ipv6_netif_addr_t *b)
+{
+    if ((a->prefix_len < 128U) && (a->prefix_len == b->prefix_len) &&
```

Comment:
The first check shouldn't be necessary. This is already checked, when the address is added to the interface.